cmake/FreecivInstall.cmake

```diff
@@ -41,10 +41,11 @@ if(WIN32 OR MSYS OR MINGW)
   # Custom command files to run the applications
   install(
     FILES
-    ${CMAKE_SOURCE_DIR}/dist/freeciv21-server.cmd
     ${CMAKE_SOURCE_DIR}/data/icons/128x128/freeciv21-client.ico
     ${CMAKE_SOURCE_DIR}/data/icons/128x128/freeciv21-modpack.ico
     ${CMAKE_SOURCE_DIR}/data/icons/128x128/freeciv21-server.ico
```

Comment:
FWIW, the icons are Windows specific too

Reply:
I'm contemplating creating them on the fly with `icotool`, but I think this would go beyond the scope of this PR and shouldn't be backported.

Reply:
Actually I just realized that these are in the icon dir and not dist so no need to do anything.